README.md

```diff
@@ -1,188 +1,194 @@
 # Ibis
 
-[![Documentation Status](https://img.shields.io/badge/docs-docs.ibis--project.org-blue.svg)](http://ibis-project.org)
-[![Project Chat](https://img.shields.io/badge/zulip-join_chat-purple.svg?logo=zulip)](https://ibis-project.zulipchat.com)
-[![Anaconda-Server Badge](https://anaconda.org/conda-forge/ibis-framework/badges/version.svg)](https://anaconda.org/conda-forge/ibis-framework)
+[![Documentation status](https://img.shields.io/badge/docs-docs.ibis--project.org-blue.svg)](http://ibis-project.org)
```

Comment:
Any particular reason to change the case? These don't show up anywhere in the code and are really only for documenting what the link is. You could just as well remove the `Documentation Status` text and keep the inner `[]` empty.

Reply:
no it just bothered me